stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/previewdata/PreviewReactionData.kt

```diff
@@ -1,45 +1,34 @@
 package io.getstream.chat.android.compose.previewdata
 
-import androidx.compose.runtime.Composable
-import androidx.compose.ui.res.painterResource
-import io.getstream.chat.android.compose.R
-import io.getstream.chat.android.compose.state.reactionoptions.ReactionOptionItemState
+import io.getstream.chat.android.client.models.Reaction
 
 /**
  * Provides sample reactions that will be used to render component previews.
  */
+@Suppress("MemberVisibilityCanBePrivate")
```

Comment:
Any reason why we have to supress this? Maybe it makes sense to just mark them off as private and open up to our needs?

Reply:
Yes, making the fields private